content/en/docs/concepts/cluster-administration/api-priority-and-fairness.md

```diff
@@ -0,0 +1,138 @@
+---
+title: API priority and fairness
+content_template: templates/concept
+weight: 20
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+The control plane of Kubernetes is itself a resource (broadly
+construed) to which access is managed.  In particular, each API server
+applies some prioritization and fairness in its handling of requests.
+This includes not only kube-apiservers but aggregated API servers too.
+
+Prioritization and fairness applies only to requests that are not
+_long-running_.  The nature of the request, not the size of its
+reply, determines whether a request is long-running.  The requests
+considered to be long running are
+[WATCHES](https://kubernetes.io/docs/reference/using-api/api-concepts/#efficient-detection-of-changes),
+[EXECs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#exec),
+and
+[fetching of logs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#logs)
+(with or without "following").
+
+Each API server independently applies prioritization and fairness,
+based on API objects that configure this feature.  This involves two
+concepts: priority level and flow schema.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Enabling API priority and fairness
+
+API priority and fairness is an alpha feature, so it is disabled by
+default.  To enable API priority and fairness you need to do two
+things: turn on the RequestManagement feature gate (e.g., by including
+the `--feature-gates RequestManagement=true` flag when starting your
+API servers) and enable the flowcontrol.apiserver.k8s.io/v1alpha1 API
+version (e.g., by including the `--runtime-config
+flowcontrol.apiserver.k8s.io/v1alpha1=true` flag when starting your
+`kube-apiserver`s).
+
+Enabling the feature gate replaces the "max in flight" filter, which
+enforces simple aggregate concurrency limits, with a filter that
+implements API priority and fairness --- which can be seen as a more
+general version of the "max in flight" filter.
+
+## Priority level
+
+Each API server has an aggregate concurrency limit and divides
+it&mdash;without examining the request load&mdash;among several
+priority levels.  The priority levels operate independently of each
+other.  That is, the API server classifies a given request to exactly
+one priority level and, for a given priority level, handles at most
+the corresponding number of requests at any given time.  One special
```

Comment:
Maybe start “One special level” on a new paragraph, or reword to make the meaning clear.

The priority level is named `exempt`, right? That's not just a description. If someone localizes these docs it needs to be clear that `exempt` is an opaque term and not a concept being introduced.